server/functions/index.js

```diff
@@ -1,19 +1,22 @@
 // The Cloud Functions for Firebase SDK to create Cloud Functions and setup triggers.
```

Comment:
Can format indents with VsCode

Reply:
I think it was already formatted using Intellij with the node.js plugin, let me know if you find any inconsistency with the indentation